source/level_set_base.cc

```diff
@@ -525,7 +525,11 @@ LevelSetBaseAlgorithm<dim>::output_solution(const std::string  output_name,
                 {
                   if (joint_fe.system_to_base_index(i).first.second == 0)
                     joint_solution(local_joint_dof_indices[i]) =
+#if true
```

Comment:
By looking here, I realize that this code still does not know that `DataOut` can take several `DoFHandler` objects living on the same triangulation. (A function I added to deal.II in 2013 or 2014, I think.) We should clean this up at some point.

Reply:
I was thinking about the same. But thought about postponing this to a later PR.